cudaungappedextender/samples/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# Enable formatting
+gw_enable_auto_formatting("${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Again, the `gw_enable_auto_formatting("${CMAKE_CURRENT_SOURCE_DIR}")` in the `cudaungappedextender/` directory should suffice.